### PR TITLE
Upgrading to NSB 7

### PR DESCRIPTION
**Pull request by HEskandari**
Still a couple of things to update:

- Bumping up the version
- Updating .nuspec

### CODE REVIEW

**Comment by SimonCropp** [comment]
i cant see the nuspec in the diff?

**Comment by HEskandari** [comment]
@SimonCropp Version updated to v4 beta.

**Comment by SimonCropp** [comment]
@HEskandari can u deploy a beta nuget?

**Comment by HEskandari** [comment]
v4.0 beta released.

**Comment by HEskandari** [comment]
Updated to latest NSB Beta 8. Released beta 3.

**Comment by HEskandari** [comment]
@SimonCropp I've pushed a commit that updates to v7 RTM. Can you verify? Can we publish the official v4?

File: NServiceBus.Mailer/NServiceBus.Mailer.csproj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (SimonCropp) [inline NServiceBus.Mailer/NServiceBus.Mailer.csproj:1]: u should update to an sdk project

File: Sample/packages.config [hunk 2]
```diff
@@ -1,4 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="NServiceBus" version="6.1.1" targetFramework="net46" />
+  <package id="Newtonsoft.Json" version="11.0.1" targetFramework="net452" />
+  <package id="NServiceBus" version="7.0.1" targetFramework="net452" />
+  <package id="NServiceBus.Newtonsoft.Json" version="2.1.0" targetFramework="net452" />
+  <package id="System.ValueTuple" version="4.4.0" targetFramework="net46" />
```
Comment (SimonCropp) [inline Sample/packages.config]: value tuple should not be required?

Comment (HEskandari) [reply Sample/packages.config]: Not sure why it was added. I'll see if I can remove.

File: NServiceBus.Mailer/NServiceBus.Mailer.csproj (hunk 1 above)
Comment (HEskandari) [reply NServiceBus.Mailer/NServiceBus.Mailer.csproj:1]: @SimonCropp Okay to do that in a separate PR? I'm anxious to get this out.

**Comment by HEskandari** [comment]
@SimonCropp I've removed the unused package. Do you think we can merge this? I'll create a separate PR to update the project format.

File: NServiceBus.Mailer/NServiceBus.Mailer.csproj (hunk 1 above)
Comment (SimonCropp) [reply NServiceBus.Mailer/NServiceBus.Mailer.csproj:1]: sounds reasonable

**Comment by SimonCropp** [comment]
LGTM

**Pull request merged**